src/js/helpers/graph/declarationHelper.js

```diff
@@ -86,7 +86,10 @@ const createVirtualEndNode = (composite, transitions) => ({
 });
 
 const createTransitionsToSubtasks = (subtasks) => subtasks
-  .reduce((total, current, index) => ({ ...total, [`_subtask_${index}`]: current }), {});
+  .reduce((total, current, index) => ({
```

Comment:
btw. for large number of subtasks this is going to be extremely slow.
This is more efficient `Object.fromEntries(subtasks.map((task,index) => [`_subtask_${index}`, task]))` 

https://www.richsnapp.com/article/2019/06-09-reduce-spread-anti-pattern

Reply:
I let myself create a small benchmark:
![image](https://user-images.githubusercontent.com/60001414/81170574-37f92880-8f9b-11ea-8b7d-2cd99349b941.png)
![image](https://user-images.githubusercontent.com/60001414/81170717-70006b80-8f9b-11ea-87f6-a40aec7ce68c.png)

Reply:
Nice, didn't know about `fromEntries` function